style.css

```diff
@@ -254,3 +264,224 @@ textarea:hover {
+@media screen and (min-width: 768px) {
+  * {
+    margin: 0;
+    padding: 0;
+  }
+
+  .main {
+    background: url(images/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-color: #fff;
+    border-bottom-left-radius: 150px;
+    width: 1440px;
+  }
+
+  body {
+    background-color: #f7f7f9;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-between;
+    padding: 20px 190px 50px 90px;
+    background-color: #fff;
+  }
+
+  .nav-links {
+    display: flex;
+  }
+
+  .nav-links ul {
+    display: flex;
+    list-style: none;
+    gap: 35px;
+  }
+
+  .nav-links ul li a {
+    color: #344563;
+    text-decoration: none;
+    font-size: 15px;
+  }
+
+  .logo {
+    font-size: 17px;
+  }
+
+  h1 {
+    font-size: 40px;
+  }
+
+  .fa-bars {
+    visibility: hidden;
+  }
+
+  .presentation,
+  .presentation2 {
+    font-size: 18px;
+    padding-left: 0;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    width: 920px;
+    height: 480px;
+    margin: 120px 0 10px 300px;
+  }
+
+  .lang {
+    padding-top: 0;
+    padding-bottom: 0;
+  }
+
+  .connect {
+    font-size: 16px;
+    padding-left: 0;
+  }
+
+  .icons ul li a {
+    font-size: 20px;
+  }
+
+  .icons {
+    margin-left: 0;
+    margin-top: -19px;
+    padding: 12px;
+  }
+
+  .works1,
+  .works2,
+  .works3,
+  .works4 {
+    height: 496px;
+    display: flex;
+    width: 1050px;
+    border-radius: 26px;
+    gap: 20px;
+    padding: 0 40px;
+    background-color: #fff;
+    margin-left: auto;
+    margin-right: auto;
+    align-items: center;
+  }
+
+  .body {
+    margin: 60px;
+    display: grid;
+    gap: 40px;
+  }
+
+  .part1,
+  .part2,
+  .part3,
+  .part4 {
+    margin-top: -100px;
+  }
+
+  .button {
+    padding: 13px;
+    width: 171px;
+  }
+
+  .button:hover {
+    background-color: #6070ff;
+  }
+
+  .photo1,
+  .photo3 {
+    width: 544px;
+    height: 448px;
+  }
+
+  .photo2,
+  .photo4 {
+    width: 544px;
+    height: 448px;
+    order: 1;
+  }
+
+  .About {
+    background-color: #fff;
+    border-top-right-radius: 100px;
+    width: 100%;
+    display: flex;
+    gap: 10px;
+    padding: 0 90px;
+    height: 700px;
+  }
+
+  .content2 {
+    width: 50%;
+    margin-left: 100px;
+    line-height: 24px;
+  }
+
+  .langs {
+    width: 550px;
+    margin-top: -12px;
+    margin-left: 0;
+    padding: 80px 70px;
+  }
+
+  .presentation2 {
+    font-size: 16px;
+  }
+
+  .lang-nav ul li {
+    margin: 20px;
+  }
+
+  .lang-nav ul {
+    margin-left: -400px;
+  }
+
+  .js {
+    display: inline-flex;
+    flex-direction: column;
+    width: 100px;
+    flex-wrap: nowrap;
+    margin: 4px;
+  }
+
+  .contact {
+    background-image: url(images/contact\ form\ background\ shapes.svg);
+    background-repeat: no-repeat;
+    background-color: #6070ff;
+    border-top-left-radius: 120px;
+    display: grid;
+    align-items: center;
+    margin-top: -100px;
+    height: 800px;
+  }
```

Comment:
[OPTIONAL] Please 🙏🏽 Fix the position of the contact section content, it is centered in the Figma design. 👇🏽

Figma design:
![image](https://user-images.githubusercontent.com/20537712/236645155-a5792668-875d-43b4-8860-f28c82a4dd78.png)

Your design :
![image](https://user-images.githubusercontent.com/20537712/236645178-90105618-e5c3-4cf2-9216-bf577c0ee0c2.png)